es.json

```diff
@@ -535,8 +534,9 @@
   "tarado": 2,
   "tarugo": 2,
   "tereso": 2,
-  "tetas": 2,
   "tetas grandes": 2,
+  "tetas": 2,
+  "tetona" 2,
```

Comment:
Looks like a syntax error here!

Reply:
Oof, fixed. I duplicated words with accents and then typed it without accents, is it ok now?